jvb/src/main/kotlin/org/jitsi/videobridge/stats/callstats/CallstatsService.kt

```diff
@@ -58,6 +57,8 @@ class CallstatsService(
     val statsTransport: StatsTransport?
         get() = callstatsTransport
 
+    val config = CallstatsConfig.config
```

Comment:
Is this needed?

Reply:
I didn't think an import of something defined within the same file would work. But it seems to work, so I replaced it.